src/main.cpp

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2020 YADRO
+
+#include "printer.hpp"
+
+#include <getopt.h>
+
+#include <iostream>
+
+/**
+ * @brief Print help usage info.
+ *
+ * @param[in] app - application's file name
+ */
+static void printHelp(const char* app)
+{
+    std::cout << "Print BMC inventory list." << std::endl;
```

Comment:
I like the `printf` and I was sure that you like too.
Besides, the OpenBMC asks to avoid `iostream` using:
https://github.com/openbmc/docs/blob/master/cpp-style-and-conventions.md#iostream

Reply:
Yes, I prefer `printf` ;)
But it is not a logging output, so I believe we can use c++ style here.
If you insist, I can replace iostream with printf.

Reply:
I think it should be done just because it looks prettier for us :)

Reply:
Done!